manifests/03_operator.cr.yaml

```diff
@@ -6,3 +7,6 @@ metadata:
     release.openshift.io/create-only: "true"
 spec:
   managementState: Managed
+  version: 4.0.0
+  logLevel: "4"
+  replicas: 1
```

Comment:
are these really existing in the API ? look odd.

Reply:
I'm assuming this is legacy. I'm just copying over from the deleted `06_config.yaml` since its subsequent application was overriding this one. Since these fields do not seem to be present in operator CRs in other repos, I'll assume it can be removed.

Reply:
Removed.